lingetic-nextjs-frontend/app/page.tsx

```
@@ -1,10 +1,317 @@
+  role: string;
+}) {
+  return (
+    <div className="p-8 border border-gray-100 hover:shadow-lg transition-shadow duration-300">
+      <div className="mb-6 text-[#2563eb]">
+        <svg
+          width="45"
+          height="36"
+          viewBox="0 0 45 36"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+        >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)